lib/bundler/templates/newgem/newgem.gemspec.tt

```diff
@@ -1,7 +1,5 @@
 # coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
-$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require '<%=config[:namespaced_path]%>/version'
+require_relative 'lib/<%=config[:namespaced_path]%>/version'
```

Comment:
My understanding is that this is a template for new gems generated with ```bundle gem bundlname``` so i'd use the phrase _requires extra work_ vs _break things_.

I'd argue this is not common for version.rb to include from lib. I've checked on my installed gems:
```sh
find ~/.rvm/gems/ruby-2.3.0/gems/ -name 'version.rb' | wc -l
486
grep -ir '\brequire\b' ~/.rvm/gems/ruby-2.3.0/gems/ --include="version.rb"
```
show that such gems in my system are:
- heroku-3.42.22
- rbnacl-3.3.0
- sass-3.2.19

i'd say in spec cases it's needed the author should be able modify the gemspec.